src/rawtoaces_util/image_converter.cpp

```diff
@@ -794,6 +848,18 @@ void ImageConverter::init_parser( OIIO::ArgParse &arg_parser )
         [&]( OIIO::cspan<const char *> argv ) { settings.verbosity++; } );
 }
 
+/// Helper function to extract keys from a map into a vector
+template <typename MapType>
+std::vector<std::string> get_map_keys( const MapType &map )
+{
+    std::vector<std::string> keys;
+    for ( const auto &pair: map )
+    {
+        keys.push_back( pair.first );
+    }
+    return keys;
+}
+
```

Comment:
I might be missing something. Not sure if there is an easier way of extracting keys from a Map.

Reply:
killing it anyway.